cmd/skupper/skupper.go

```diff
@@ -852,20 +701,77 @@ func init() {
 	cmdVersion := NewCmdVersion(newClient)
 	cmdDebugDump := NewCmdDebugDump(newClient)
 
+	//backwards compatibility commands hidden
+	deprecatedMessage := "please use 'skupper service [bind|unbind]' instead"
+	cmdBind.Hidden = true
+	cmdBind.Deprecated = deprecatedMessage
+	cmdUnbind.Deprecated = deprecatedMessage
+	cmdUnbind.Hidden = true
+
+	cmdListConnectors := NewCmdListConnectors(newClient) //listconnectors just keeped
+	cmdListConnectors.Hidden = true
+	cmdListConnectors.Deprecated = "please use 'skupper link status all'"
```

Comment:
Why don't we assume `all` as the default and use like `--name name` if we need specific info?
Asking this in order to avoid possible (malicious) clash between `all` (the special flag) and `all` literal link name.

Reply:
I agree with Fernando.  In earlier discussion, we specified this:

skupper link status [<name>]

Where having a name means it's restricted to one link, and not having a name means get status for all.

Reply:
you are correct, the "all" is there because it comes from the deprecated "check-connection" but it seems it can be easily fixed, will push one more commit in minutes

Reply:
https://github.com/skupperproject/skupper/pull/360/commits/5cba87c0735e9b9c9c805b881c667a38d4b65cc0